Makefile

```diff
@@ -21,6 +21,11 @@ ANDROIDNDKVER := 21.4.7075529
 
 SDK := ${ANDROID_HOME}/android-sdk-$(PLATFORM)
 
+ADB := adb
+DOCKER := docker
```

Comment:
Since we're handling `DOCKER` as an environment variable and not just a make variable, I think it might be nice to change this to `DOCKER ?= docker`. Then you could do `export DOCKER=podman` in your shell and not have to pass it in every time.